src/core/userDataHelper.js

```diff
@@ -30,6 +33,35 @@ var userDataHelper = {
       return userId === team.userid;
     });
   },
+  /**
+   * Get the list of tagged words for a user's notes. Memoize result.
+   * @param userid
+   * @param notes
+   * @returns {Array}
+   */
+  getWordbankWords: function(userid, notes) {
```

Comment:
works for now / demo but as you mentioned in your email it will reside in the api wrapper (and also the mock api wrapper in this instance). 

Given the time available I would rather see using the wordFreqs and also onclick functionality when selecting a tag that we could attach other esatereggs too with some of the pre-seeded tags (as and idea :)